src/code/components/allele-filters.js

```diff
@@ -1,4 +1,4 @@
-import {PropTypes} from 'react';
+import React, {PropTypes} from 'react';
```

Comment:
What's the purpose of this when we've turned off the `react-in-jsx-scope` rule? If this is preparing for a time when `React` might not be a global, should we turn back on the rule?